### PR TITLE
Improve representation of attachments in issues

### PR DESCRIPTION
**Pull request by mschoettle**
Improves the representation of attachments in issues. Credit goes to @renevdzee (https://github.com/go-gitea/gitea/issues/6089#issuecomment-545035188) for the initial patch.

This fixes issues #6501 and #6089.

The attachments are moved into the content and separated with a divider. Contrary to the original patch I removed the image thumbnail because it was in place of the file size. I tried to replace the icon on the left with the image thumbnail but it's quite big (35px vs. 16px).

attachments_divider

What would be nice (is there any way to do that?) if the icon is mime-type related. Right now this is only supported for images.

Alternatively, the attachments could remain in the bottom as shown below. It requires a few CSS changes:

attachments_bottom

### CODE REVIEW

File: templates/repo/issue/view_content.tmpl [hunk 1]
```diff
@@ -55,20 +55,19 @@
 						</div>
 						<div id="comment-{{.Issue.ID}}" class="raw-content hide">{{.Issue.Content}}</div>
 						<div class="edit-content-zone hide" data-write="issue-{{.Issue.ID}}-write" data-preview="issue-{{.Issue.ID}}-preview" data-update-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/content" data-context="{{.RepoLink}}" data-attachment-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/attachments" data-view-attachment-url="{{$.RepoLink}}/issues/{{.Issue.Index}}/view-attachments"></div>
+						{{if .Issue.Attachments}}
+							<div class="ui clearing divider"></div>
+                            <div class="ui middle aligned padded grid">
+								{{template "repo/issue/view_content/attachments" Dict "ctx" $ "Attachments" .Issue.Attachments}}
+							</div>
```
Comment (zeripath) [inline templates/repo/issue/view_content.tmpl]: Missing a closing `<div>` here and the indenting is different from the rest of the file.

File: templates/repo/issue/view_content/comments.tmpl [hunk 2]
```diff
@@ -52,20 +52,19 @@
 					</div>
 					<div id="comment-{{.ID}}" class="raw-content hide">{{.Content}}</div>
 					<div class="edit-content-zone hide" data-write="issuecomment-{{.ID}}-write" data-preview="issuecomment-{{.ID}}-preview" data-update-url="{{$.RepoLink}}/comments/{{.ID}}" data-context="{{$.RepoLink}}" data-attachment-url="{{$.RepoLink}}/comments/{{.ID}}/attachments"></div>
+					{{if .Issue.Attachments}}
+							<div class="ui clearing divider"></div>
+                            <div class="ui middle aligned padded grid">
+								{{template "repo/issue/view_content/attachments" Dict "ctx" $ "Attachments" .Issue.Attachments}}
+							</div>
```
Comment (zeripath) [inline templates/repo/issue/view_content/comments.tmpl]: Similarly missing a closing `<div>` here and the indenting is different from the rest of the file.

Comment (mschoettle) [reply templates/repo/issue/view_content/comments.tmpl]: Indentation should be fixed now. I can't see where there should be a missing `</div>`, though.

File: templates/repo/issue/view_content.tmpl (hunk 1 above)
Comment (mschoettle) [reply templates/repo/issue/view_content.tmpl]: Indentation should be fixed now. I can't see where there should be a missing `</div>`, though.

**Review by zeripath — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by zeripath** [comment]
@mschoettle I think when you created this PR you must have denied project mergers the ability to update the branch in order to merge this. 

Please either re-allow project mergers to update the branch or update the branch - however, if you do not allow project mergers to update the branch there is a significant chance this will delay our ability to merge this PR.

**Comment by mschoettle** [comment]
@zeripath Sorry about that. I was wondering what this setting was useful for. I enabled "Allow edits from maintainers".

**Comment by guillep2k** [comment]
ping lg-tm

**Comment by guillep2k** [comment]
I was waiting **so much** for this PR. Thanks! ❤️

File: templates/repo/issue/view_content/comments.tmpl [hunk 3]
```diff
@@ -52,20 +52,19 @@
 					</div>
 					<div id="comment-{{.ID}}" class="raw-content hide">{{.Content}}</div>
 					<div class="edit-content-zone hide" data-write="issuecomment-{{.ID}}-write" data-preview="issuecomment-{{.ID}}-preview" data-update-url="{{$.RepoLink}}/comments/{{.ID}}" data-context="{{$.RepoLink}}" data-attachment-url="{{$.RepoLink}}/comments/{{.ID}}/attachments"></div>
+				{{if .Issue.Attachments}}
```
Comment (lunny) [inline templates/repo/issue/view_content/comments.tmpl:55]: This should display comment's attachements but not issue's

**Pull request merged**